### PR TITLE
Add mage and update footer to remove gradio branding

### PR DESCRIPTION
**Pull request by Rochan-HM**

### CODE REVIEW

File: css/evaluation.css [hunk 1]
```diff
@@ -1,139 +1,144 @@
...
 .tooltip:hover .tooltiptext {
-    visibility: visible;
-    opacity: 1;
+  visibility: visible;
+  opacity: 1;
+}
+
+footer {
+  visibility: hidden;
 }
```
Comment (Rochan-HM) [inline css/evaluation.css:144]: Oops... Not sure why the diff is so massive.
I think some issue with LF and CRLF?
Or maybe black/prettier auto-formatted the code?

But this should be the only change here - we are hiding the footer.

Comment (cs1090218) [reply css/evaluation.css:144]: Yeah its just auto formatted (probably from some vscode extension). Let me know I'll add that extension as well.

**Review by cs1090218 — APPROVED** [review]
Thanks!
Did you plan to add the MAGE key in SKILL_NAME_TO_MCM_URL in constants.py in this PR?

**Comment by Rochan-HM** [comment]
> Thanks! Did you plan to add the MAGE key in SKILL_NAME_TO_MCM_URL in constants.py in this PR?

Yes that’s the plan :)
I think that should be sufficient 
Right now mage also returns additional metadata such as token usage. But we can refactor the “response” key to be just the answer and maybe “metadata” to be all the metadata

**Pull request merged**